dask_ml/model_selection/_incremental.py

```diff
@@ -111,15 +118,22 @@ def _create_model(model, ident, **params):
 def _fit(
     model,
     params,
-    X_train,
-    y_train,
-    X_test,
-    y_test,
+    X_train: da.Array,
+    y_train: da.Array,
+    X_test: da.Array,
+    y_test: da.Array,
     additional_calls,
     fit_params=None,
     scorer=None,
     random_state=None,
+    log_freq: float = 1.0,
```

Comment:
log_freq -> log_frequency here and elsewhere.

Is this a standard in scikit-learn? Or are we starting a new pattern with log_freq.

Reply:
Now `verbose: Union[bool, int]` print `verbose` percent of the time if specified as an integer. That satisfies my constraint that higher values of `verbose` print more often.